rabbit-escape-engine/src/rabbitescape/engine/solution/SolutionInterpreter.java

```diff
@@ -45,9 +46,20 @@ public SolutionInterpreter( Solution solution, boolean appendWon )
         this.command = null;
         this.untilCount = -1;
     }
+    
+    public static SolutionInterpreter getNothingPlaying()
+    {
+        SolutionInterpreter nothingPlaying = new SolutionInterpreter( SolutionParser.parse( "1" ) );
+        nothingPlaying.emptySteps = true;
+        return nothingPlaying;
+    }
```

Comment:
I have not written it yet, but when the solution runs out of steps, and the level has not finished the level needs to change to emptySteps true. So I did not make it final.

continuePastEnd does not sound right for a SolutionInterpreter that starts empty (everything apart from demo mode).